Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation-overrides.json

```diff
@@ -4,6 +4,6 @@
     },
     "-webkit-appearance": {
         "description": "Changes the appearance of buttons and other controls to resemble native controls.",
-        "syntax": "none | button | caret | checkbox | default-button | inner-spin-button | listbox | listitem | media-play-button | media-slider | media-sliderthumb | media-volume-slider | media-volume-sliderthumb | menulist | menulist-button | meter | progress-bar | progress-bar-value | push-button | radio | searchfield | searchfield-cancel-button | searchfield-decoration | searchfield-results-button | searchfield-results-decoration | slider-horizontal | slider-vertical | sliderthumb-horizontal | sliderthumb-vertical | square-button | textarea | textfield | -apple-pay-button"
+        "syntax": "none | button | checkbox | default-button | listbox | media-play-button | media-slider | media-sliderthumb | media-volume-slider | media-volume-sliderthumb | menulist | menulist-button | meter | progress-bar | push-button | radio | searchfield | slider-horizontal | slider-vertical | square-button | textarea | textfield | -apple-pay-button"
```

Comment:
Only updating the `-overrides.json` file won't make the change appear in Web Inspector's UI. You need to also regenerate `CSSDocumentation.js` using `Source/WebInspectorUI/Scripts/update-inspector-css-documentation` (or manually apply your changes to `CSSDocumentation.js` as well). Updating this file just ensure that future runs of the script won't undo your edits.